### PR TITLE
Fixes crash in Graphics Benchmarks caused by null pointer.

### PR DESCRIPTION
**Pull request by munish-b**
regression with AIA – Github build at OpenCI. The GFXBench
and GLBenchmark apks were crashing

Jira: None.
Test: Execution of GFXBench and GLBenchMark is okay
Signed-off-by: Munish Bhardwaj<munish.bhardwajx@intel.com>

### CODE REVIEW

File: src/util/build_id.c [hunk 1]
```diff
@@ -114,7 +114,12 @@ build_id_find_nhdr_for_addr(const void *addr)
 unsigned
 build_id_length(const struct build_id_note *note)
 {
-   return note->nhdr.n_descsz;
+   unsigned length = 0;
+
+   if(note)
+      length = note->nhdr.n_descsz;
+
+   return length ;
```
Comment (tpalli) [inline src/util/build_id.c]: return note ? note->nhdr.n_descsz : 0;

**Comment by munish-b** [comment]
updated

**Comment by codernavi18** [comment]
@tpalli 
This issue caused a regression in AIA and hence waiting for merge.

**Comment by kalyankondapally** [comment]
Hmm I cannot reproduce this issue, u tested with latest mesa here  ?

**Comment by tpalli** [comment]
I cannot reproduce this issue either.

**Comment by tpalli** [comment]
I've pushed a different workaround for this that also fixes issues with Vulkan and program binary extensions. Thanks!

**Pull request closed without merging**